webapp/src/views/Console/DomainDetailsDialog.vue

```diff
@@ -24,7 +24,8 @@
       </v-alert>
       <v-card-text>
         <p>
-          To properly secure your domain with DNSSEC, please forward the following information to your domain registrar:
+          Please forward the following information to the administrator/registry of
```

Comment:
For most TLDs, domain owners cannot interact directly with the registry and instead have to talk to their registrar or provider. I would say:

"Please forward the following information to the organization where you registered the domain <code>...</code> (usually your provider or technical administrator):"

Reply:
done